### ISSUE TITLE
[Arcs] Can't be raided

### ISSUE THREAD

**Issue by turmoilbyrd** [LederCards/cards#70]
[Bunkers](https://cards.ledergames.com/card/ARCS-F1803) and [Banners](https://cards.ledergames.com/card/ARCS-F1003) are harmed like buildings but "can't be raided". There is some confusion on what exactly that means.

Current consensus is that this is based on the fact they are not true buildings, and should operate on the Raid Dice Limit (https://arcs.seiyria.com/#7.7.1.3.1-raid-dice-limit and https://arcs.seiyria.com/#20.4.3.1-raid-dice-limit). Therefore:

- If the owner has Bunker/Banner and no other building/Flagship in the same system, the attacker cannot collect Raid dice
- If the owner has Bunker/Banner and some other building/Flagship in the same system, the attacker can now collect Raid dice. The Building Hit rolled can be used to hit Bunker/Banner
- If the owner of Bunker/Banner has no other building/Flagship in any systems on the map, then the attacker can again collect Raid dice

Are those reasoning correct?

**Comment by turmoilbyrd** [comment]
related #71

**Comment by jdyearsley** [comment]
Yes, that reasoning is correct! Is there a compelling alternative reading that should be FAQ'ed to warn against?

**Comment by turmoilbyrd** [comment]
> Yes, that reasoning is correct! Is there a compelling alternative reading that should be FAQ'ed to warn against?

Basically in the discussion, some were wondering if "cannot" means a bunker would, say, shield another city in the same system from being raided

**Comment by jdyearsley** [comment]
Cool, I will add an FAQ addressing that question.
